test/suite/test_bug019.py

```diff
@@ -72,10 +78,10 @@ def populate(self, nentries):
     # assert a file is created within 90 seconds.
     def prepfiles(self):
         for i in range(1,90):
-                f = fnmatch.filter(os.listdir('.'), "*Prep*")
-                if f:
-                        return f
-                time.sleep(1.0)
+            f = fnmatch.filter(os.listdir('.'), "*Prep*")
+            if f:
+                return
+            time.sleep(1.0)
         self.assertFalse(not f)
```

Comment:
This double negative feels hard to parse.  I would use something like `self.fail('prep file not created')`